src/System.Net.Primitives/src/System/Net/HttpStatusCode.cs

```diff
@@ -54,15 +61,34 @@ public enum HttpStatusCode
         UnsupportedMediaType = 415,
         RequestedRangeNotSatisfiable = 416,
         ExpectationFailed = 417,
+        // Removed status code: ImATeapot = 418.
```

Comment:
Instead of saying "Removed status code"...please add a comment describing why we aren't "adding" this status code.  Please add the various links from the discussion thread on #4382.

Reply:
I.e. from the discussion thread...

>It would be a mistake to add it to .NET now. See golang/go#21326, nodejs/node#14644, requests/requests#4238 and aspnet/HttpAbstractions#915

Reply:
Ok.